core/src/main/scala/cats/arrow/Arrow.scala

```diff
@@ -45,4 +45,50 @@ import simulacrum.typeclass
   @simulacrum.op("***", alias = true)
   def split[A, B, C, D](f: F[A, B], g: F[C, D]): F[(A, C), (B, D)] =
     andThen(first(f), second(g))
+
+  /**
+    * Create a new computation `F` that merge outputs of `f` and `g` both having the same input
+    *
+    * Example:
+    * {{{
+    * scala> import cats.implicits._
+    * scala> val addEmpty: Int => Int = _ + 0
+    * scala> val multiplyEmpty: Int => Double= _ * 1d
+    * scala> val f: Int => (Int, Double) = addEmpty &&& multiplyEmpty
+    * scala> f(1)
+    * res0: (Int, Double) = (1,1.0)
+    * }}}
+    *
+    * Note that the arrow laws do not guarantee the non-interference between the _effects_ of
+    *  `f` and `g` in the context of F. This means that `f &&& g` may not be equivalent to `g &&& f`.
+    */
+  @simulacrum.op("&&&", alias = true)
+  def merge[A, B, C](f: F[A, B], g: F[A, C]): F[A, (B, C)] = {
+    andThen(lift((x: A) => (x, x)), split(f, g))
+  }
+
+  /**
+    * Create a new computation `F` that apply f andThen biforks the result.
+    * On one way it is applied to g and on the other it is passed through.
+    * The final result is a tuple
+    * <br/><pre>
+    *             /--out1 = f(input) ----\
+    *  input -->{                          }--->(out1,out2)
+    *             \--out2 = g(f(input))--/
+    *             </pre>
+    * Example:
+    * {{{
+    * scala> import cats.implicits._
+    * scala> val twoTimes: Int => Double = _ * 2d
+    * scala> val fiveTimes: Double => Double= _ * 5
+    * scala> val f: Int => (Double, Double) = twoTimes -< fiveTimes
+    * scala> f(2)
+    * res0: (Double, Double) = (4.0,20.0)
+    * }}}
+    *
+    */
+  @simulacrum.op("-<", alias = true)
+  def combineAndBypass[A, B, C](f: F[A, B], g: F[B, C]): F[A, (B, C)] = {
+    andThen(lift((x: A) => (x, x)), split(f, andThen(f, g)))
```

Comment:
One downside of this is that it calculates `f(x)` twice. I think that for pretty much any arrow you would want to override this to reuse the result of `f(x)`.

Reply:
I agree. I think I incline to not adding this one, since it's just an alias to `merge(f, f >>> g)`

Reply:
Yes you are right. It is also the same of this one:
 `andThen(andThen( f, lift((x: B) => (x, x)) ), split(id, g))`
This last one  evicts the double calculation.
 Essentially: input=> f(input) => out1=> (out1,out1)=> (out1,g(out1))=>(out1,out2)
Please consider if it's better to eliminate combineAndBypass because it is possible to use `merge(f, f >>> g)` or if it is good to have combineAndBypass as a sort of shortcut.
I'll prepare the commit following your decision.

Reply:
mmm.... I think that also `merge(f, f >>> g) ` evaluates f twice, instead this one `andThen(andThen( f, lift((x: B) => (x, x)) ), split(id, g))` evaluates f one time, so that it is not properly a simple shortcut althought the result is the same.